### PR TITLE
Update Rules: Add new loss types and clarify loss != target metric

### PR DESCRIPTION
**Pull request by fsschneider**
Added L1 and CTC loss as possible losses.

Added that the target performance might not be the loss but a different, workload-specific metric such as the accuracy.

Bumped version number.

### CODE REVIEW

File: RULES.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ model_fn(
 loss_fn(label_batch, logits_output_batch) -> 1d array of losses per example  # differentiable
 ```
 
-- Unlike in the [Model Track](#model-track), we will specify the loss function name in order to let training algorithms depend on the loss function. It will be one of {**mean squared error**, **cross-entropy**}.
+- Unlike in the [Model Track](#model-track), we will specify the loss function name in order to let training algorithms depend on the loss function. It will be one of {**mean squared error**, **cross-entropy**, **CTC**, or **L1**}.
```
Comment (georgedahl) [inline RULES.md:110]: L1 reconstruction error

File: RULES.md [hunk 2]
```diff
@@ -365,7 +365,7 @@ The target performance on each held-out workload will be defined by using the pe
...
+Submissions will be scored based on their required training time to reach the target performance of each workload. The target performance can be the same as the loss but might also be a different workload-specific metric such as the accuracy or BLEU score. The training time includes compilation times for computation graphs and ops that could happen just-in-time during training; all our benchmarks should be fast enough to compile so as not to dramatically impact overall performance. The overall ranking is then determined by summarizing the performances across all [workloads](#workloads), both public and held-out, using [performance profiles](http://www.argmin.net/2018/03/26/performance-profiles/), as explained below.
```
Comment (georgedahl) [inline RULES.md:368]: "The target performance can ..." -> "The target performance metric can ..."

**Review by georgedahl — APPROVED** [review]

File: RULES.md (hunk 2 above)
Comment (georgedahl) [reply RULES.md:368]: Probably should also change "loss" to "loss function"
